// [[Reason: TS normalizes types]] ////

//// [tests/cases/conformance/classes/members/inheritanceAndOverriding/derivedClassOverridesProtectedMembers2.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -10,37 +10,27 @@
 };
 declare class Base {
     protected a: typeof x;
     protected b(a: typeof x): invalid;
-    protected get c(): {
-        foo: string;
-    };
+    protected get c(): typeof x;
     protected set c(v: typeof x);
     protected d: (a: typeof x) => void;
     protected static r: typeof x;
     protected static s(a: typeof x): invalid;
-    protected static get t(): {
-        foo: string;
-    };
+    protected static get t(): typeof x;
     protected static set t(v: typeof x);
     protected static u: (a: typeof x) => void;
     constructor(a: typeof x);
 }
 declare class Derived extends Base {
     a: typeof y;
     b(a: typeof y): invalid;
-    get c(): {
-        foo: string;
-        bar: string;
-    };
+    get c(): typeof y;
     set c(v: typeof y);
     d: (a: typeof y) => void;
     static r: typeof y;
     static s(a: typeof y): invalid;
-    static get t(): {
-        foo: string;
-        bar: string;
-    };
+    static get t(): typeof y;
     static set t(a: typeof y);
     static u: (a: typeof y) => void;
     constructor(a: typeof y);
 }
